homeassistant/components/avea/light.py

```diff
@@ -0,0 +1,96 @@
+"""Support for the Elgato Avea lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_HS_COLOR, SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR, Light)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.util.color as color_util
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_AVEA = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Avea platform."""
+    import avea
+
+    try:
+        nearby_bulbs = avea.discover_avea_bulbs()
+        for bulb in nearby_bulbs:
+            bulb.get_name()
+            bulb.get_brightness()
+    except OSError as err:
+        raise PlatformNotReady from err
+
+    add_devices(AveaLight(bulb) for bulb in nearby_bulbs)
+
+
+class AveaLight(Light):
+    """Representation of an Avea."""
+
+    def __init__(self, light):
+        """Initialize an AveaLight."""
+        self._light = light
+        self._name = light.name
+        self._state = None
+        self._brightness = light.brightness
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_AVEA
+
+    @property
+    def name(self):
+        """Return the display name of this light."""
+        return self._name
+
+    @property
+    def brightness(self):
+        """Return the brightness of the light.
+
+        This method is optional. Removing it indicates to Home Assistant
+        that brightness is not supported for this light.
+        """
+        return self._brightness
+
+    @property
+    def is_on(self):
+        """Return true if light is on."""
+        return self._state
+
+    def turn_on(self, **kwargs):
+        """Instruct the light to turn on.
+
+        You can skip the brightness part if your light does not support
```

Comment:
Please remove this sentence. Just keep the first sentence.

Reply:
Thanks, fixed in 781d19821460edf8c6a089c9d8f8858ac0cf9462 👍